src/librustc/hir/map/collector.rs

```diff
@@ -139,9 +182,9 @@ impl<'a, 'hir> NodeCollector<'a, 'hir> {
         let node_hashes = self
             .hir_body_nodes
             .iter()
-            .fold(Fingerprint::ZERO, |fingerprint, &(def_path_hash, dep_node_index)| {
+            .fold(Fingerprint::ZERO, |fingerprint, &(def_path_hash, hash)| {
```

Comment:
Can you rename `fingerprint` to something like `combined_fingerprint` and `hash` to `fingerprint` here? That should make it easier to understand what's going on.